File: python/v6.py
import ipaddress

ip = ipaddress.ip_address('::1')
print("Language: Python")
print(f"=== Checking IP: {ip} ===")
print(f"Is Private: {ip.is_private}")
print(f"Is Loopback: {ip.is_loopback}")
print()

File: nodejs/v6.js
const ip = require('ip');

const ipAddress = '::1';
console.log("Language: Node.js");
console.log(`=== Checking IP: ${ipAddress} ===`);
console.log(`Is Private: ${ip.isPrivate(ipAddress)}`);
console.log(`Is Loopback: ${ip.isLoopback(ipAddress)}`);
console.log();
